### PR TITLE
Add conda-forge installation documentation for Linux and MacOSX

### PR DESCRIPTION
**Pull request by jessica-mitchell**
This PR adds a page to the installation guide on how to install NEST using the conda-forge nest-simulator package, setup by @steffengraber. The conda-forge package is a minimal install of nest.

I've added 'beta' to the page because this is new and testing has been limited.

Additionally, there are a few changes to the CSS style sheet for improved look of the HTML.

### CODE REVIEW

**Comment by jessica-mitchell** [comment]
I have just updated the conda instructions since we have an openmpi version of the conda-forge package.

File: doc/_static/css/custom.css [hunk 1]
```diff
@@ -98,6 +98,20 @@ code.descclassname {
 .highlight {
     background: rgba(255, 165, 0, 0.1);
 }
+<<<<<<< HEAD
```
Comment (heplesser) [inline doc/_static/css/custom.css]: This looks like something left from a merge conflict.

File: doc/installation/conda_install.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+.. code-block:: sh
+
+  conda install -c conda-forge nest-simulator
+
+or WITH openmpi:
+
+.. code-block:: sh
+
+   conda install -c conda-forge nest-simulator=*=mpi_openmpi*
```
Comment (heplesser) [inline doc/installation/conda_install.rst]: This looks like two ==, one outside and one inside *...*, unless I do not understand RST syntax right.

File: doc/installation/conda_install.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+.. note::
+
+   The current version on conda-forge is NEST 2.16.0
```
Comment (heplesser) [inline doc/installation/conda_install.rst]: May need updating soon?

File: doc/installation/conda_install.rst (hunk 2 above)
Comment (jessica-mitchell) [reply doc/installation/conda_install.rst]: The meaning behind the syntax is as follows:
`conda install -c conda-forge nest-simulator=<version>=<build_string>`

We could include that as an explanation if needed?

File: doc/installation/conda_install.rst (hunk 3 above)
Comment (jessica-mitchell) [reply doc/installation/conda_install.rst]: Yep I didn't want to leave this in as is but was thinking we should include some information about the versions available on conda-forge. Perhaps something along the lines of "The conda-forge package is available from NEST 2.16.0 onward"?

File: doc/_static/css/custom.css (hunk 1 above)
Comment (jessica-mitchell) [reply doc/_static/css/custom.css]: removed

**Comment by heplesser** [comment]
Yes, I think more explanation or an example here is needed.



--

Dr Hans Ekkehard Plesser
Associate Professor

Faculty of Science and Technology
Norwegian University of Life Sciences
PO Box 5003, 1432 Aas, Norway

Phone +47 6723 1560
Email hans.ekkehard.plesser@nmbu.no<mailto:hans.ekkehard.plesser@nmbu.no>
Home http://arken.nmbu.no/~plesser


On 7 Jun 2019, at 11:38, jessica-mitchell <notifications@github.com<mailto:notifications@github.com>> wrote:


@jessica-mitchell commented on this pull request.

________________________________

In doc/installation/conda_install.rst<https://github.com/nest/nest-simulator/pull/1204#discussion_r291521124>:

> +
+  conda activate ENVNAME
+
+3. Install the latest `nest-simulator` package from conda-forge. You can choose to install it
+
+WITHOUT openmpi:
+
+.. code-block:: sh
+
+  conda install -c conda-forge nest-simulator
+
+or WITH openmpi:
+
+.. code-block:: sh
+
+   conda install -c conda-forge nest-simulator=*=mpi_openmpi*


The meaning behind the syntax is as follows:
conda install -c conda-forge nest-simulator=<version>=<build_string>

We could include that as an explanation if needed?

**Comment by heplesser** [comment]
:+1: I approve assuming small doc fixes will come in place. I won’t be able to push the approve button for a while, so consider this my approval.

--

Dr Hans Ekkehard Plesser
Associate Professor

Faculty of Science and Technology
Norwegian University of Life Sciences
PO Box 5003, 1432 Aas, Norway

Phone +47 6723 1560
Email hans.ekkehard.plesser@nmbu.no<mailto:hans.ekkehard.plesser@nmbu.no>
Home http://arken.nmbu.no/~plesser


On 7 Jun 2019, at 11:47, jessica-mitchell <notifications@github.com<mailto:notifications@github.com>> wrote:


@jessica-mitchell commented on this pull request.

________________________________

In doc/_static/css/custom.css<https://github.com/nest/nest-simulator/pull/1204#discussion_r291524017>:

> @@ -98,6 +98,20 @@ code.descclassname {
 .highlight {
     background: rgba(255, 165, 0, 0.1);
 }
+<<<<<<< HEAD


removed

File: doc/installation/conda_install.rst (hunk 3 above)
Comment (jessica-mitchell) [reply doc/installation/conda_install.rst]:
```suggestion
   The conda-forge package is available from NEST 2.16.0 onward
```

**Comment by heplesser** [comment]
:+1:

--

Dr Hans Ekkehard Plesser
Associate Professor

Faculty of Science and Technology
Norwegian University of Life Sciences
PO Box 5003, 1432 Aas, Norway

Phone +47 6723 1560
Email hans.ekkehard.plesser@nmbu.no<mailto:hans.ekkehard.plesser@nmbu.no>
Home http://arken.nmbu.no/~plesser


On 7 Jun 2019, at 12:12, jessica-mitchell <notifications@github.com<mailto:notifications@github.com>> wrote:


@jessica-mitchell commented on this pull request.

________________________________

In doc/installation/conda_install.rst<https://github.com/nest/nest-simulator/pull/1204#discussion_r291532201>:

> +
+WITHOUT openmpi:
+
+.. code-block:: sh
+
+  conda install -c conda-forge nest-simulator
+
+or WITH openmpi:
+
+.. code-block:: sh
+
+   conda install -c conda-forge nest-simulator=*=mpi_openmpi*
+
+.. note::
+
+   The current version on conda-forge is NEST 2.16.0


⬇️ Suggested change

-   The current version on conda-forge is NEST 2.16.0
+   The conda-forge package is available from NEST 2.16.0 onward

File: doc/installation/conda_install.rst (hunk 2 above)
Comment (jessica-mitchell) [reply doc/installation/conda_install.rst]:
```suggestion
   conda install -c conda-forge nest-simulator=*=mpi_openmpi*
   
Where the syntax for this install follows the pattern: ``nest-simulator=<version>=<build_string>``
```

**Pull request merged**